### ISSUE TITLE
Metadata does not work when transcoding an audio stream

### ISSUE THREAD

**Issue by radioshock-ru** [savonet/liquidsoap#2213]
**Describe the bug**
```
set("log.stdout",true);
set("log.file",false);
set("log.level",5);
set("decoder.file_decoders",["META","WAV","AIFF","MIDI","IMAGE","OGG","MAD"]);
set("encoder.encoder.export", ["artist","title","album","genre","date","tracknumber","comment","track","year"]);
input = mksafe(input.http(timeout=5.0, "http://127.0.0.1:80/radioshock_fm", new_track_on_metadata=true));
output.icecast(%mp3(bitrate=128, samplerate=44100, stereo=true), format="audio/mpeg", icy_metadata="true", encoding="UTF-8", host="127.0.0.1", port=80, password="PASSWORD", mount="/radioshock", genre="Rock",  description="Radio of St. Petersburg, Russia", name="Radio SHOCK", url="http://www.radioshock.ru", input);
output.icecast(%mp3(bitrate=48),format="audio/mpeg", icy_metadata="true", encoding="UTF-8", host="127.0.0.1", port=80, password="PASSWORD", mount="/radioshock48", genre="Rock",  description="Radio of St. Petersburg, Russia", name="Radio SHOCK", url="http://www.radioshock.ru", input);
```
There is Metadata on the source stream (title). After transcoding the audio stream, metadata is displayed by default, and the dynamic one is not updated from the source.

Tell me where to look for the problem?

I apologize for the bad English, I use transliteration.

**Version details**
 - OS: [Debian, 11]
 - Version [`2.1.0`]

**Comment by toots** [comment]
Thanks for this report. There are ongoing changes in `2.1.x` that could have created this issue. Our roadmap at the moment is to first fix issues coming up with the `2.0.x` branch and get back to `2.1.x`. Thanks for your patience and feel free to checkout the `v2.0.4-preview` upcoming stable release if you need!

**Comment by toots** [comment]
I believe that this is closed now. Feel free to reopen if needed.
